Example_Projects/SwiftCoAPServerExample/SwiftCoAPServerExample/SCMessage.swift

```diff
@@ -34,7 +34,7 @@ protocol SCCoAPTransportLayerDelegate: class {
 
 protocol SCCoAPTransportLayerProtocol: class {
     //SCClient uses this property to assign itself as delegate
-    weak var transportLayerDelegate: SCCoAPTransportLayerDelegate! { get set }
+    var transportLayerDelegate: SCCoAPTransportLayerDelegate! { get set }
```

Comment:
Same as above?

Reply:
Xcode Warning:
`'weak' should not be applied to a property declaration in a protocol and will be disallowed in future versions`
![](https://i.ibb.co/4PmkS48/Screenshot-2019-12-24-at-02-28-01.png)